src/esp/physics/CollisionGroupHelper.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "CollisionGroupHelper.h"
+#include <Corrade/Utility/Assert.h>
+#include "esp/core/logging.h"
+
+namespace esp {
+namespace physics {
+
+/*
+  enum class CollisionGroup {
+    Default = 1,
+    Static = 2,
+    Kinematic = 4,
+    FreeObject = 8,
+    GraspedObject = 16,
+    Robot = 32,
+  };
+*/
+
+int CollisionGroupHelper::getMaskForGroup(CollisionGroup group) {
```

Comment:
Two problems with this implementation:
1. It is app-specific and shouldn't be hard-coded in hab-sim (as I've already mentioned)
2. As far as actually specifying collision-group interaction (whether in C++ or python), a better way to do this is to explicitly enable interaction between pairs of groups. A problem with the current approach here is that you are able to express contradictory interactions: for example, in Static's case statement, Static can be set to interact with GraspedObject, while in the GraspedObject case statement, GraspedObject might be set to *not* interact with Static.